src/utils/get-organized-arguments.ts

```diff
@@ -287,12 +287,26 @@ export default async function getOrganizedArguments(): Promise<OrganizedArgument
 
 		// Validate data, if necessary
 		if (mergedSpec.data.accepts && mergedSpec.data.accepts instanceof Array) {
-			// @ts-expect-error: TypeScript is confused here...
-			if (!mergedSpec.data.accepts.includes(organizedArguments.data)) {
+			// Find unrecognized data by removing acceptable values
+			let remnants = ` ${organizedArguments.data} `;
+
+			for (const current of mergedSpec.data.accepts) {
+				const find = ` ${current} `;
+				const foundAt = remnants.indexOf(find);
+
+				if (foundAt >= 0) {
+					remnants = remnants.substr(0, foundAt) + remnants.substr(foundAt + find.length - 1);
```

Comment:
Since there are other uses of template literals here, couldn't this line be one as well?

Reply:
Hmmm.  Well, I suppose it can be simplified a little bit.